public/scss/authentication.scss

```diff
@@ -57,4 +57,12 @@ body.authentication {
   form[name="gateway_cancel_second_factor_verification"] {
     text-align: center;
   }
+
+  .form-error {
+    margin-bottom: 1em;
```

Comment:
Nitpick, but to be consistent with other Ibuildings projects it's best to put these properties in alphabetical order.

Reply:
Also with EB / SPD

Reply:
Ah, now I  understand, the properties are to be sorted alphabetically. :+1: